src/compare-oracle-to-fallback/agent-setup.js

```diff
@@ -0,0 +1,103 @@
+const ethers = require('ethers');
+
+const {
+  Finding, FindingSeverity, FindingType, getJsonRpcUrl,
+} = require('forta-agent');
+
+// load required shared types
+const {
+  LendingPoolAddressesProvider: lendingPoolAddressesProvider,
+  ProtocolDataProvider: protocolDataProviderAddress,
+} = require('../../contract-addresses.json');
+const { abi: protocolDataProviderAbi } = require('../../abi/AaveProtocolDataProvider.json');
+const { abi: aaveOracleAbi } = require('../../abi/AaveOracle.json');
+const { abi: priceOracleGetterAbi } = require('../../abi/IPriceOracleGetter.json');
+const { abi: lendingPoolAddressesProviderAbi } = require('../../abi/ILendingPoolAddressesProvider.json');
+const { aaveEverestId } = require('../../agent-config.json');
+
+// set up the an ethers provider
+// do not use ethers.providers.WebSocketProvider in production (there is no support)
+const jsonRpcProvider = new ethers.providers.JsonRpcProvider(getJsonRpcUrl());
+
+function calculatePercentError(first, second) {
+  const delta = first.minus(second).absoluteValue();
+  return delta.div(first).multipliedBy(100);
+}
+
+// helper function to create alerts
+function createAlert(reserveToken, tokenPrice, tokenPriceFallback, percentError, tokenAlert) {
+  const { numAlertsInLastDay } = tokenAlert;
+  return Finding.fromObject({
+    name: `Large AAVE Price Oracle / Fallback Oracle Difference for ${reserveToken.symbol}`,
```

Comment:
Looking at the Forta explorer I suggest we make the alert name more concise.  How about:
"Aave Fallback Oracle Price Difference for <token>"

Also use "Aave" not "AAVE"

Reply:
Done and done.